radio/src/gui/480x272/model_setup.cpp

```diff
@@ -636,8 +647,11 @@ bool menuModelSetup(event_t event)
            IF_ACCESS_MODULE_RF(EXTERNAL_MODULE, 0),   // Receiver 1
            IF_ACCESS_MODULE_RF(EXTERNAL_MODULE, 0),   // Receiver 2
            IF_ACCESS_MODULE_RF(EXTERNAL_MODULE, 0),   // Receiver 3
-
-         TRAINER_ROWS
+           
+         TRAINER_ROWS,
+//OW
+         MAVLINK_ROWS
```

Comment:
perhaps you could write directly  LABEL(Mavlink), 0, 0, 0, 0 (no need of a define here, even VC++ shouldn't cry)

Reply:
sure. I think I was just trying to follow patterns :)

Reply:
done. thx.